CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
-# Changelog for unreleased
+# Changelog for 2.0.0
 
-The following sections list the changes for unreleased.
+The following sections list the changes for 2.0.0.
 
```

Comment:
This file will be auto generated by calens, no need to. bump version here.

Reply:
just me not trusting the `machines`. I always run the changelog locally. But won't do it next time :)